test/jdk/sun/security/tools/keytool/ReadJar.java

```diff
@@ -23,7 +23,7 @@
 
 /**
  * @test
- * @bug 6890872 8168882 8257722
+ * @bug 6890872 8168882 8257722 8255552
```

Comment:
Here we are just updating the warning message, so I don't think the bugid needs to be added.

Reply:
Ok.